.github/workflows/toc-review-reminder.yaml

```diff
@@ -1,59 +0,0 @@
-# Copyright 2024 The Knative Authors.
```

Comment:
Maybe disable the  job - might be worth discussing if steering wants to continue this practice